deploy-px-k8s/etcd.md

```diff
@@ -17,7 +17,7 @@ docker run -d --net=host -p 4001:2379 \
    --initial-cluster node1=http://${IP}:2380
 ```{{execute T1}}
 
-For production setups, follow [this page](https://docs.portworx.com/maintain/etcd.html) for recommendations on etcd.
+For production setups, follow [Portworx docs](https://docs.portworx.com/) for the recommendations on etcd. As of Version 2.0 you have the option to use a built in etcd.
```

Comment:
@bascar Update the newly added etcd doc link here.

Reply:
Updated